apps/acqdat_core/lib/acqdat_core/streams/schema/logic_chain.ex

```diff
@@ -0,0 +1,43 @@
+defmodule AcqdatCore.Schema.Streams.LogicChain do
+  @moduledoc """
+  Allows to interconnect logic chains.
+  """
+  use AcqdatCore.Schema
+
+  @typedoc """
+  """
+  @type t :: %__MODULE__{}
+
+  @primary_key false
+  @derive {Jason.Encoder, only: [:pipeline_id]}
+  embedded_schema do
+    field(:pipeline_id, :string)
+  end
+
+  @all_params ~W(pipeline_id)a
+
+  @doc """
+  Creation changeset for `logic_chain` action.
+  """
+  @spec create_changeset(t | Ecto.Changeset.t(), map) :: Ecto.Changeset.t()
+  def create_changeset(action, params) do
+    action
+    |> changeset(params)
+  end
+
+  @doc """
+  Update changeset for `logic_chain` action.
+  """
+  @spec update_changeset(t | Ecto.Changeset.t(), map) :: Ecto.Changeset.t()
+  def update_changeset(action, params) do
+    action
+    |> changeset(params)
```

Comment:
Please avoid single piping
```suggestion
   changeset(action, params)
```